slurm/nccl-test-distributed-a100-64.slurm

```diff
@@ -9,7 +9,7 @@
 #SBATCH --exclusive # Use --exclusive to ensure no other jobs run on the same nodes
 
 
-
+. /usr/share/modules/init/bash
 module load image-defaults
```

Comment:
In my environment, `module` was not on the `PATH` of the compute nodes. This adds the standard setup command to initialize `modules` in a shell, which resolves that.